application/controllers/api/Feed.php

```diff
@@ -86,10 +86,10 @@ public function authors_get()
 		}
 	}
 
-	public function lastest_releases_get()
+	public function latest_releases_get()
```

Comment:
This changes the API path, right?

Before: `https://librivox.org/api/feed/lastest_releases`
After: `https://librivox.org/api/feed/latest_releases`

(I don't know who uses that API, sorry.)

Reply:
Yep, good catch. I had to go back and add https://github.com/LibriVox/librivox-catalog/commit/148ab02ee6b51550b8b6c643464bd46878244cb5 to keep backwards compatibility with whatever was using that misspelled endpoint in the wild.